### ISSUE TITLE
Does not seem to support the use of @graph

### ISSUE THREAD

**Issue by rogargon** [semantifyit/sdo-check#5]
SDO Check does not find any markup if @graph is used. For instance, as in the following example:
https://gist.github.com/walkergv/dd942c4bd33267821f17bd4f618ce86f

The old https://search.google.com/structured-data/testing-tool does work in this case. Is this no longer acceptable though it is proper JSON-LD?

**Comment by OmarJAH** [comment]
Hola Roberto,

The tool focuses primarily on the JSON-LD format that is used commonly for schema.org (uses no @graph, @reverse and such), see the goals section of https://github.com/semantifyit/sdo-check/wiki

Anyway, with the latest commits, the tool also detects and renders some annotations using @graph but the verification process and rendering of the errors in the visualization are still not correct in such cases.

TLDR: Support of @graph is a feature we want to have at some point, but it has not the highest priority at the moment.
